.pdd

```diff
@@ -4,6 +4,6 @@
 --exclude target/**/*
 --exclude src/main/resources/images/**/*
 --exclude .idea/**/*
---rule min-words:20
+--rule min-words:15
```

Comment:
How this change is related to this Pull Request?

Reply:
@Sammers21 it's needed to allow submitting smaller puzzles than 20 words, see todos in `pom.xml`

Reply:
But the Pull Request name is `Bump jdk-11`, which is not related to puzzlers in any way